core/Azure.Mcp.Core/src/Areas/Server/Commands/ServiceStartCommand.cs

```diff
@@ -466,6 +466,13 @@ private IHost CreateHttpHost(ServiceStartOptions serverOptions)
         });
 
         // Configure services
+        // Update telemetry configuration to set the transport mode
```

Comment:
This feels disjoint from all of the other Azure*Server related configuration.  This could go into ConfigureOpenTelemetry with the other telemetry enabling/disabling logic

Reply:
This looks like an incomplete draft, but the original discussion was to just use the existing if-def that we're temporarily using for HTTP / docker for the next 2 weeks.  This is a short term point in time solution which will be reverted next month.  If we can do this as a ~3 line change just for the docker build flavor that would be ideal.